BookMyShow.Data/City.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace BookMyShow.Data;
+
+public partial class City
```

Comment:
I asked you to create separate layers. one for database and one for business layer. Please segregate it properly.